src/index.ts

```diff
@@ -0,0 +1,102 @@
+export const booleanCalculator = (input:string):boolean | void => {
+    let evaluatedStatement = [];
+    input = formatInput(input);
+    
+    let inputArray: string[] = input.split(" ");
+    evaluatedStatement = evaluator(inputArray);
+    if (input == "NOT TRUE" || input == "FALSE") {
+        return false;
+    } else if (input == "NOT FALSE" || input == "TRUE") {
+        return true;
+    } else if (evaluatedStatement[0] == "TRUE") {
+        return true;
+    } else if (evaluatedStatement[0] == "FALSE") {
+        return false;
+    }
+}
+
+const formatInput = (input:string):string => {
```

Comment:
Its a smart idea to add spaces around brackets, I wish I thought of this. Makes handling brackets much nicer